src/AspNet.Security.OAuth.Validation/Events/MessageReceivedContext.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Http;
 
-namespace AspNet.Security.OAuth.Validation.Events
+namespace AspNet.Security.OAuth.Validation
 {
-    public class ReceivingTokenContext : BaseControlContext
+    public class MessageReceivedContext : BaseControlContext
```

Comment:
The reason for the name change was mainly to follow what the `JwtTokenMiddleware` was doing. Essentially it's saying "hey, a (request) message showed up" and not just a token necessarily, even though that's all we process with this middleware right now.